ovirt/resource_ovirt_vm.go

```diff
@@ -261,6 +276,15 @@ func resourceOvirtVM() *schema.Resource {
 					},
 				},
 			},
+			"type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: fmt.Sprintf(
```

Comment:
Hi @rgolangh , shall we define an extra `ValidateFunc` function to validate enum properties for fail-fast? Thanks.
```golang
    ValidateFunc: validation.StringInSlice([]string{
        string(ovirtsdk4.VMTYPE_DESKTOP),
        string(ovirtsdk4.VMTYPE_SERVER),
        string(ovirtsdk4.VMTYPE_HIGH_PERFORMANCE),
    }, false),
```

Reply:
by all means.

Reply:
ack

Reply:
@rgolangh thank you.